metadata_writer_inventory_conn.c

```diff
@@ -56,28 +56,30 @@ static int32_t md_inventory_execute_insert_update(struct md_writer_sqlite *mws,
         sqlite3_bind_int(stmt, 5, mce->sequence) ||
         sqlite3_bind_int(stmt, 6, mce->l3_session_id) ||
         sqlite3_bind_int(stmt, 7, mce->l4_session_id) ||
-        sqlite3_bind_text(stmt, 8, mce->event_value_str, strlen(mce->event_value_str), SQLITE_STATIC) ||
-        sqlite3_bind_int(stmt, 9, mce->interface_type) ||
-        sqlite3_bind_text(stmt, 11, mce->network_address, strlen(mce->network_address), SQLITE_STATIC)){
+        sqlite3_bind_int(stmt, 8, mce->has_ip) ||
+        sqlite3_bind_int(stmt, 9, mce->connectivity) ||
+        sqlite3_bind_int(stmt, 10, mce->connection_mode) ||
+        sqlite3_bind_int(stmt, 11, mce->quality) ||
+        sqlite3_bind_int(stmt, 12, mce->interface_type) ||
+        sqlite3_bind_text(stmt, 14, mce->network_address, strlen(mce->network_address), SQLITE_STATIC)){
```

Comment:
Is all of the changes made to this file "safe"? I.e., will they create the same SQL as before (including the csv)?